requirments.txt

```diff
@@ -1,2 +1,3 @@
+asteval==0.9.19
```

Comment:
Does it need to be pinned to a particular version or can we saw newer than 0.9
```suggestion
asteval>=0.9.19
```

Reply:
this is optional comment, feel free to ignore

Reply:
I generally prefer to be specific in requirements.txt (tells you how to recreate a complete, working environment) and flexible in setup.py (tells pip what dependencies need to be picked up if e.g. installing in some other environment). PyPA have a [few notes](https://packaging.python.org/discussions/install-requires-vs-requirements/) on the matter.